### PR TITLE
Update documentation and deployment information

### PR DESCRIPTION
**Pull request by Gehock**
# Description

Documentation changes made during radar.minus.cs.aalto.fi and radar-focal.cs.aalto.fi deployment.

## Is it [Done](https://wiki.aalto.fi/display/EDIT/Definition+of+Done)?

*Clean up your git commit history before submitting the pull request!*

- [x] I (developer) have tested the changes manually
- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly

### CODE REVIEW

**Comment by Gehock** [comment]
I'm not sure if I was planning on adding something extra to it, but looks ready enough for me as is

File: README.md [hunk 1]
```diff
@@ -1,155 +1,168 @@
...
-	Install software
+    sudo apt-get install git libjpeg-dev
+    sudo apt-get install libxml2-dev libxslt-dev zlib1g-dev
 
-		git clone https://github.com/Aalto-LeTech/mooc-grader.git
-		mkdir mooc-grader/uploads
+Install software
 
-2. ### Python requirements (2.7 should work too)
+    git clone https://github.com/Aalto-LeTech/mooc-grader.git
```
Comment (Mankro) [inline README.md:31]: The URL is nowadays https://github.com/apluslms/mooc-grader.git (organization apluslms)

File: README.md [hunk 2]
```diff
@@ -1,155 +1,168 @@
...
-		sudo pip3 install virtualenv
 
-	Then, create virtual environment with grader requirements.
+Then, create virtual environment with grader requirements.
 
-		virtualenv -p python3 venv
-		source venv/bin/activate
-		pip install -r mooc-grader/requirements.txt
+    python3 -m venv venv
+    source venv/bin/activate
```
Comment (Mankro) [inline README.md:42]: Side note: someone once told me that `source` and `.` would behave differently. To my understanding, they are the same and `source` is an alias in Bash whereas `.` is the standard name (in POSIX?). What do you think?

**Review by Mankro — APPROVED** [review]
Nice!

File: README.md (hunk 1 above)
Comment (Mankro) [reply README.md:31]: I fixed this myself now.

File: README.md (hunk 2 above)
Comment (Gehock) [reply README.md:42]: The [Bash manual](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html) describes `source` as "A synonym for .". To my knowledge the behaviour is identical

**Pull request merged**